core/shared/src/main/scala/com/monovore/decline/HelpFormat.scala

```diff
@@ -0,0 +1,27 @@
+package com.monovore.decline
+
+sealed abstract class HelpFormat extends Product with Serializable {
```

Comment:
I'd be tempted to define this on the `Help` companion so folks can refer to `Help.Format` and `Help.Plain` etc., but I don't feel so strongly.